db/migrate/20211111121403_create_boards.rb

```diff
@@ -0,0 +1,19 @@
+class CreateBoards < ActiveRecord::Migration[6.1]
+  def change
+    create_table :boards do |t|
+      t.string :name, null: false
+      t.integer :visibility, default: 0
```

Comment:
Maybe consider a boolean field?

Reply:
3 options on boolean? Good luck, it's not gonna be used for now anyways.

Reply:
Thought it will be only 2 options. So maybe an enum?